[Home](https://andre-ye.github.io) > [Biology Navigation](https://andre-ye.github.io/biology/biology_navigation) > [Biology Content](https://andre-ye.github.io/biology/biology_navigation#biology-content) > Study Guides > Biology Cheat Sheet / Reference

# Biology Cheat Sheet / Reference
Andre Ye, 11/7/2020

*This biology cheat sheet will be handy reference for assignments, discussions, and test prep. It will be continually updated as the course progresses.*

---

## Navigate
- [Terms and Concepts](#terms-and-concepts)
  * [DNA Replication](#dna-replication)
  * [Transcription and Translation](#transcription-and-translation)
- [Core Ideas](#core-ideas)
  * [Chemistry, Origin of Life, Nucleic Acids](#chemistry--origin-of-life--nucleic-acids)
    + [Chemical Foundation of Life Terms and Concepts](#chemical-foundation-of-life-terms-and-concepts)
    + [Two Theories](#two-theories)
    + [RNA vs DNA Levels of Structure](#rna-vs-dna-levels-of-structure)
    + [3 DNA Replication Hypotheses](#3-dna-replication-hypotheses)
  * [Proteins](#proteins)
    + [Protein Levels of Structure](#protein-levels-of-structure)
    + [Protein Functions and Types](#protein-functions-and-types)
    + [Proteins Required for Synthesis in Bacteria](#proteins-required-for-synthesis-in-bacteria)
  * [Translation, Transcription, and the Genetic Code](#translation--transcription--and-the-genetic-code)
    + [Complete Genetic Code Table](#complete-genetic-code-table)
    + [Properties of the Genetic Code](#properties-of-the-genetic-code)
    + [Types of Point Mutations](#types-of-point-mutations)
    + [The 3 Categories of Mutations](#the-3-categories-of-mutations)
    + [Process of Initiating Transcription in Bacteria](#process-of-initiating-transcription-in-bacteria)
    + [Process of Ending Transcription in Bacteria](#process-of-ending-transcription-in-bacteria)
    + [Processed mRNA Strand Diagram](#processed-mrna-strand-diagram)
    + [Process of Protein Synthesis in Ribosomes](#process-of-protein-synthesis-in-ribosomes)
    + [Process of Initiating Translation in Bacteria](#process-of-initiating-translation-in-bacteria)
    + [Process of Elongation Phase of Translation](#process-of-elongation-phase-of-translation)
    + [Process of Terminating Translation](#process-of-terminating-translation)
    + [Complete Process of the Central Dogma](#complete-process-of-the-central-dogma)
    + [Central Dogma in Bacteria vs Eukaryotes](#central-dogma-in-bacteria-vs-eukaryotes)
- [Important Experiments](#important-experiments)
  * [Miller-Urey Experiment](#miller-urey-experiment)
  * [Herschey-Chase Experiment](#herschey-chase-experiment)
  * [Meselson-Stahl Experiment](#meselson-stahl-experiment)
  * [Srb-Horowitz Experiment](#srb-horowitz-experiment)
  * [Zamecnik et al Experiment](#zamecnik-et-al-experiment)
    
---

## Terms and Concepts
### DNA Replication

| Term | Definition |
| --- | --- |
| Chromosome theory of inheritance | Proposed that chromosomes contain genes. |
| Nitrogen 14 | Normal nitrogen; used in the Meselson-Stahl experiment to distinguish parent strands from daughter strands. |
| Nitrogen 15 | Heavy nitrogen with one more neutron; used in the Meselson-Stahl experiment to distinguish parent strands from daughter strands. |
| 5' to 3' Direction | The direction in which DNA is synthesized. |
| Antiparallel strands | Strands of DNA that lie opposite each other. |
| Complementary base pairing | Base A matched with base T, base G matches with base C. |
| Density gradient centrifugation | Separates molecules based on their density. Lower-density molecules cluster in bands high in the centrifuge tube, higher-density molecules cluster in bands lower in the tube. |
| dNTP | DNA synthesis requires an input of energy, but potential energy of deoxyribonucleotide monomers is raiased by reactions tha add two phosphate groups, forming deoxyribonucleotide triphosphates (dNTPs). dNTP has high potential energy; forms phosphodiester bonds in DNA strand. |
| Discontinuous Replication | Proposed to explain how lagging strand was synthesized. Primase synthesizes new RNA primers for lagging strands as the moving replication fork opens single-stranded regions of DNA; DNA polymerase syntheses short DNA fragments from these primers. Fragments linked to form a continuous strand. |
| Lagging strand | The strand that runs opposite to the direction in which the replication fork is moving. This causes a lag. |
| Continuous strand | The strand that moves in the same direction as the replication fork. |
| Okazaki fragments | Short DNA fragments attached to RNA primers produced by the lag of the lagging strnad. |
| Origin of Replication | A sequence of bases that indicates the start of a replication bubble, in which two replication forks move in opposite directions to separate the DNA strand. | 
| Replisome | Proteins and enzymes work in a macromolecular machine called the replisome. Replisome may contain up to three copies of DNA polymerase III.
| RNA polymerase | A polymerase that can start synthesis from scratch. Primase is a RNA polymerase (enzymes that catalyze polymerization of ribonucleotides into RNA). |
| RNA primers | Because DNA polymerase cannot synthesize new nucleotides on its onw, but RNA polymerase (for example, primase) can. This is used to initiate building, and is later removed by DNA polymerase I. |

### Transcription and Translation

Term | Definition
--- | ---
mutation | The altering of genes of chromosomes in some sort of way.
codon | A group of three nucleotide bases that specifies an amino acid.
fitness | In the context of transcription and translation, the ability for a gene expression to positively impact the phenotype such that the genotype gets passed down to another generation.
frameshift | Mutations that shift the reading frame. These are usually point addition or deletions that affect the position of every base after it.
gene expression | The process by which gene information is used to produce a gene product.
genetic code | The mapping of each codon to an amino acid.
messenger RNA (mRNA) |
mutants | Genes whose sequence or chromosomes that have been altered in some sort of way.
reading frame | The sequence of codons, the sequence broken down into groups of three beginning at the start codon and ending at the end codon.
RNA polymerase | A polymerase that can synthesize RNA without a primer, unlike DNA polymerase.
start codon | A codon that indicates when protein synthesis should begin. This is `AUG`.
stop codon | Indicate when protein synthesis should stop. There are three: `UAA`, `UAG`, and `UGA`.
transcription | The transferring of genetic information from DNA to forms of RNA.
translation | The transferring of information from RNA to proteins.
translocation | A chromosomal error in which a broken piece of a chromosome is attached to a different chromosomes.
triplet code | A three-base code (forming codons).
3' poly(A) tail | Enzyme cuts the 3’ end of the pre-MRNA after the poly(A) signal, and specialized RNA polymerase adds 100 to 250 adenine nucleotides, forming this. It is not encoded by the DNA template strand, yet required for ribosomes to start translation and to protect the end of mRNA from attack by enzymes.
5' cap | An added cap consisting of odified guanine nucleotide linked to transcript in an unusual way when the 5' end of pre-mRNA emerges. Enables ribosomes to bind to the mRNA and protects the 5’ end of the mRNA from enzymes that degrade RNA (ribonucleases).
aminoacyl tRNA | THe amino acid attached to the transfer RNA.
aminoacyl/A site | The first site an aminoacyl tRNA diffuses to in ribosomal protein synthesis. If its anticodon matches the codon in mRNA, it stays in the ribosome.
anticodon | The codon attached to tRNA that is the base-pair complement of a codon in the mRNA. For example, the anticodon of `AAA` is `UUU`.
AUG codon | The one start codon.
coding strand | The strand opposite to the one used during transcription to build the complementary pre-mRNA strand (template strand). The transcripted pre-mRNA is synthesized in the same direction as the coding strand. 
downstream | The RNA polymerase moves in this direction when moving along the DNA.
elongation | Occurs after RNA polymerase leaves the promoter region as it synthesizes RNA, during which the enzyme reads the DNA template on the 3’ end at a rate of 50 nucleotides per second.
exit/E site | The last of three sites through which tRNA passes through, in which the uncharged tRNA is ejected from the ribosome after dropping off its amino acid.
exons | Regions that are transcribed and represented in final RNA (**ex**pressed in mature DNA).
introns | Regions of a gene that are transcribed but not represented in the final RNA.
peptidyl/P site | The second of three sites through which tRNA passes through, in which the amino acid attached to a tRNA forms a peptide bond with the existing polypeptide.
phosphorylation | The introduction of phosphate groups after a protein has been synthesized, which drastically increases the activity of the protein.
polyribosomes | Formed in bacteria when ribosomes attach to mRNA and begin synthesizing proteins before transcription was complete. Polyribosomes increase the number of copies of a protein that can be made from single mRNA.
pre-mRNA | Eukaryotic genes initially copied from nonfunctional RNAs. Same as 'primary RNA transcript'.
primary RNA transcript | Eukaryotic genes initially copied from nonfunctional RNAs. Same as 'pre-mRNA'.
promoters | Sites where transcription should begin.
protein folding | Impacts how the protein functions. Folding is often facilitated by molecular chaperones.
ribosomal RNA | Combined with proteins, these make up ribosomes. rRNA is found in ribosomes and constitutes most of the active site of ribosomes, which are considered ribozymes.
ribosome | A component of the cell outside of the nucleus that is responsible for reading mRNA and synthesizing an appropriate protein.
RNA processing | Is used to convert primary transcripts (pre-mRNA) into mature and functional RNA.
RNA splicing | Is the removal and rejoining of introns from a transcription. Done with snRNPs (small nuclear ribonucleoproteins).
spliceosomes | An aggregate macromolecular machine consisting of snRNPs (small nuclear ribonucleoproteins).
template strand | The strand used during transcription to build the complementary pre-mRNA strand.
termination | The termination of a process is when it ends.
transfer RNA (tRNA) | tRNA is the 'intermediate' step between mRNA and amino acids. Each tRNA has an amino acid has on the 3' end of the molecule a site for amino acid attachment and on the opposite side an anticodon to attach to the mRNA.
upstream | DNA is upstream of the point of reference. The RNA polymerase moves in the opposite direction of this when moving along the DNA.
wobble hypothesis | Most amino acids are specified by more than one codon. Codons for the same amino acid often have the same nucleotides for the first and second positions but different nucleotides in the third position. Certain bases bind to bases that do not match Watson-Crick base pairing, producing flexibility ('wobble') in base pairing. This allows tRNA to read more than one codon. Wobble pairing does not account fo redundancy.

---

## Core Ideas

### Chemistry, Origin of Life, Nucleic Acids

#### Chemical Foundation of Life Terms and Concepts

| Term | Definition |
| --- | ------- |
| Atom | the smallest identifiable unit of matter. |
| Element | a substance that consists entirely of a single type of atom. |
| Isotope | forms of an element with different numbers of neutrons. |
| Atomic Number | characteristic number of protons in an atom. |
| Mass Number | sum of protons and neutrons in an atom. |
| Covalent Bond | when two atoms share electrons, and the connected atoms form a molecule. Make the atoms more stable, often complete shells. |
| Compound | atoms of different elements are bonded together. |
| Electronegativity | when atoms of different elements bond, they may pull electrons towards their nuclei with different strengths. |
| Nonpolar Covalent Bond | a bond that involves equally shared electrons. |
| Polar Covalent Bond | a bond that involves assymetrically shared electrons. |
| Ionic Bond | electrons in ionic bonds are completely transferred from one atom to another. |
| Solvent | an agent for dissolving, or getting substances into solution. |

#### Two Theories

| Theory | Process |
| --- | ------------------ |
| Prebiotic Soup Model | Simple molecules like `N2`, `NH3`, and `CO` were present in the atmosphere of ancient earth. Energy in sunlight drove reactions among simple molecules to produce molecules like `formaldehyde` and `hydrogen cyanide`. Stimulated by heat, the products formed more complex molecules like `ribose`, `glycine`, and `acetaldehyde`. |
| Surface Metabolism Model | Simple molecules like `N2`, `NH3`, and `CO` were present in early oceans and hydrothermal events. Vent minerals catalyzed spontaneous reactions among high-energy molecules to produce, for instance, `acetic acid`. Concentration and heat formed more complex molecules like ribose.

#### RNA vs DNA Levels of Structure

| Level of Structure | DNA | RNA |
| --- | --- | --- |
| Primary | Sequence of deoxyribonucleotides: bases are A, T, G, C | Sequence of ribonucleotides: bases are A, U, G, C |
| Secondary | Two antiparallel strands twist into a double helix, stabilized by hydrogen bonding, hydrophobic bonding, hydrophobic interactions, and van der Waals interactions | Single strand folds back on itself to form a double-helical 'stem' and an unpaired 'loop'. |
| Tertiary | Double helical DNA forms compact structures by wrapping around histone proteins or twisting into supercoils. | Secondary structures fold to form a wide variety of distinctive three-dimensional shape. |

#### 3 DNA Replication Hypotheses

Hypothesis | Description
--- | ---
Semiconservative replication | If parental strands of DNA separate, each could be used as a synthesis of a new daughter strand. Each daughter DNA molecule consists of one old strand and one new strand. Conserves only one of the strands.
Conservative replication | If bases of strands turned out from the helix, they could serve as a template for an entirely new double helix all at once.
Dispersive replication | Parental double helix was fragmented into small pieces before replication, and each piece was replicated either with conservative or semiconservative mechanisms. Fragments would be joined into two molecules that contained a mixture of parental and daughter strands.


### Proteins

#### Protein Levels of Structure

| Level of Structure | Description | Stabilized by |
| --- | --- | --- |
| Primary | Each protein has a unique primary structure, determined by the number and sequence of amino acids, making up the polypeptide chain. 20 amino acids are used to build proteins. Various amino acids could be linked in almost any sequence. | Peptide bonds |
| Secondary | Parts of the polypeptide chain are folded or coiled. For example, forms alpha helix - chain twists forms a helix or beta pleated sheats - chain folds back on itself, or two regions lie parallel. Results from hydrogen bonding between atoms of the polypeptide backbone. | Hydrogen bonding between groups along the peptide-bonded backbone |
| Tertiary | Superimposed on primary and secondary structure: irregular loops and folds that give the protein its 3d shape. Results from interactions along R groups - hydrophilic or polar `R`-groups may hydrogen bond with each other or turn outwards and bond with surrounding water and hydrophobic or nonpolar `R` groups cluster on the inside of the protein, away from water. | Bonds and other interactions between R-groups or between R-groups and a peptide-bonded backbone; sulfur-containing strong covalent bonds |
| Quaternary | Some proteins are made up multiple polypeptide chains. Results from combination of two or more polypeptide | Same interactions that stabilize tertiary structure. |

#### Protein Functions and Types

Type | Functions
--- | ---
Structural Proteins | Anchors cell parts; serves as tracks along which cell parts can move. Binds cells together to form muscles, ligaments, etc.
Signal Proteins | Hormonal proteins that coordinate an organism's activity by acting as a signal between cells.
Transport Proteins | Carry molecules from place to place.
Sensory Proteins | Detect environmental signals like light.
Enzyme Proteins | A protein that changes the rate of a chemical reaction without itself being changed into a different molecule in the process.
Storage Proteins | Stockpile materials used to make other proteins. Store nutrient and energy-rich molecules for later use.
Contractile (Motor) Proteins | Move parts of a cell.
Gene Regulatory Proteins | Bind to DNA in particular locations and control whether certain genes will be read.
Defensive Proteins | Defensive proteins help organisms fight infection, heal damaged tissue, and evade predators.

#### Proteins Required for Synthesis in Bacteria

| Purpose | Name | Function |
| --- | --- | --- |
| Opening the helix | Helicase | Catalyzes the separation of DNA strands to open the double helix. Breaks hydrogen bonds. |
| Opening the helix | Single-strand DNA-binding Proteins (SSBPs) | Stabilizes single-stranded DNA. |
| Opening the helix | Topoisomerase | Breaks and rejoins the DNA double helix by breaking phosphodiester bonds to relieve twisting forces caused by the opening of the helix. |
| Leading-strand synthesis | Primase | Catalyzes the synthesis of the RNA primer. |
| Leading-strand synthesis | DNA Polymerase III | Extends the leading strand. |
| Leading-strand synthesis | Sliding clamp | Holds DNA in place during strand extention. |
| Lagging-strand synthesis | Primase | Catalyzes the synthesis of the RNA primer on the Okazaki fragment. |
| Lagging-strand synthesis | DNA Polymerase III | Extends an Okazaki fragment. |
| Lagging-strand synthesis | Sliding clamp | Holds DNA polymerase in place during strand extention. |
| Lagging-strand synthesis | DNA Polymerase I | Removes the RNA primer and replaces it with DNA. |
| Lagging-strand synthesis | DNA ligase | Catalyzes the jooining of Okazaki fragments into a continuous strand. |

### Translation, Transcription, and the Genetic Code
#### Complete Genetic Code Table
![Complete Genetic Code](https://andre-ye.github.io/biology/genes-and-dna/codon%20base%20table.PNG)

#### Properties of the Genetic Code

| Property | Description |
| --- | --- |
| Redundant | All amino acids (except for 2) are coded for by more than one codon. Codons specifying the same amino acid are synonymous codons. |
| Unambiguous | A given codon never codes for more than one amino acid. | 
| Non-overlapping | Once the ribosome locks onto the first codon, the reading frame is established and the ribosome rads one codon one after another. |
| Universal | Witha f ew exceptions, codons specify the same amino acids in all organisms. |
| Conservative | Several codons specify the same amino acid, but the first two bases are usually identical. |
- If a change in DNA sequence leads to a change in the third position, it is less likely to alter the amino acid in the protein.
- Genetic code minimizes the phenotypic affects of small alterations.
  - Genetic code not assembled randomly; honed by natural selection and is remarkably efficient.
  
#### Types of Point Mutations
![Types of point mutations](https://andre-ye.github.io/biology/genes-and-dna/types%20of%20point%20mutations%20table.PNG)

#### The 3 Categories of Mutations
1. **Beneficial.** Some mutations increase the fitness of an organism.
2. **Neutral.** If the mutation has not effect on fitness, it is neutral. e.g. silent mutations.
3. **Deleterious.** Most individuals are well-adapted to their current habitat and mutations are random changes in genotype; most mutations lower fitness.

#### Process of Initiating Transcription in Bacteria
1. Initiation begins. Sigma binds to the promoter region of the DNA, characterized by the -35 box and the -10 box.
2. Initiation continues. RNA polymerase opens up the DNA helix and transcription begins. NTPs are used to create the complement of the template strand. 
3. Initiation completes. Sigma is released from the core enzyme and RNA synthesis continues. The RNA polymerase moves downstream along the DNA.

#### Process of Ending Transcription in Bacteria
1. Hairpin forms. RNA polymerase transcribes a transcription-termination signal, which codes for RNA that forms a hairpin. 
2. Termination. The RNA hairpin leads to the RNA separating from RNA polymerase, terminating transcription.

#### Processed mRNA Strand Diagram
![](https://andre-ye.github.io/biology/genes-and-dna/pre%20mrna%20diagram.PNG)

#### Process of Protein Synthesis in Ribosomes
 1. Aminoacyl tRNA diffuses into the A site. If its anticodon matches a codon in mRNA, it stays in the ribosome.
 2. A peptide bond forms between the amino acid held by the aminoacyl tRNA in the A site and the growing polypeptide in the P site.
 3. The ribosome moves relative to the mRNA by one codon. All three tRNAs are shifted one position within the ribosome.
   - tRNA in E site exits.
   - tRNA in P site moves to the E site.
   - tRNA in A site switches to P site.
   - A site is empty and ready to accept another aminoacyl tRNA.

#### Process of Initiating Translation in Bacteria
1. **mRNA binds to a small subunit.** A sequence in mRNA called the ribosome binding site binds to a complementary seuqnece in an RNA molecule, which is part of the small subunit of the ribosome. This is helped by initiation factors.
2. **Initiator aminoacyl tRNA binds to the start codon.** This usually carries f-Met.
3. The **large subuit of the ribosome binds**, completing the ribosome assembly; translation can begin.

#### Process of Elongation Phase of Translation
1. **Incoming aminoacyl tRNA** moves into the A site. Its anticodon base pairs with the mRNA codon.
2. **Peptide bond formation**. The amino acid attached to the tRNA in the P site is transferred by formation of a peptide bond to the amino acid of the tRNA in the A site.
3. **Translocation**. The ribosome moves one codon down the mRNA with the help of elongation factors. THe tRNA attached to the polypeptide moves into the P site. The A site is empty.
4. **Incoming aminoacyl tRNA**. A new charged tRNA moves into the A site, where its anticodon base-pairs with an mRNA codon.
5. **Peptide-bond formation.** The polypeptide chain attached to the tRNA in the P site is transferred by peptide bond formation to the aminoacyl tRNA in the A site.
6. **Translocation.** THe ribosome moves one codon down the mRNA. The tRNA attached to the polypeptide chain moves into the P site. Uncharged tRNA from P site moves to the E site, where tRNA is ejected. The A site becomes empty again.

#### Process of Terminating Translation
1. **Release factor binds to stop codon.** When the translocating ribosome reaches a stop codon, a protein release factor fills up the A site and breaks the bond linking tRNA to the polypeptide chain.
2. **Polypeptide and uncharged tRNA are released.**
3. **Ribosome subunits separate.** They are ready to attach tot he start codon of another message.

#### Complete Process of the Central Dogma
1. Through transcription, DNA stored in the nucleus of the cell is used via RNA polymerase to create pre-mRNA.
2. Through RNA processing, the primary transcript (pre-mRNA), through which introns are removed and caps & tails are added to produce *mature mRNA*.
3. Outside of the nucleus in the cytoplasm, the mature mRNA is translated to produce a polypeptide.
4. Post-translational modifications, including folding, glycosylation, and phosphorylation are made to make the polypeptides formed by translation functional.

#### Central Dogma in Bacteria vs Eukaryotes

Process | Aspect | Bacteria | Eukaryotes
--- | --- | --- | --- |
Transcription | RNA polymerase | One | Three, each produces a different class of RNA.
Transcription | Promoter structure | Contains a -35 box and a -10 box | Variable and larger, often includes a TATA box about -30 from the transcription start site.
Transcription | Proteins that associate with promoter | Sigma and variants | General transcription factors |
RNA processing | | Rare | Extensive, several steps occur before RNA is exported to the cytoplasm.
Translation | Initiation and termination | Less complex | More complex
Translation | Elongation | Similar to eukaryotes | Similar to bacteria

---

## Important Experiments

### Miller-Urey Experiment

| Aspect | Description |
| --- | -------------------- |
| Question | Can simple molecules and kinetic energy lead to chemical evolution? |
| Hypothesis | Chemical evolution of organic molecules will occur in environments simulating early Earth conditions. |
| Null Hypothesis | Chemical evolution will not occur in early Earth simulations. |
| Prediction of Hypothesis | If kinetic energy is added to a mix of simple molecules, complex organic compounds will be produced. |
| Prediction of Null Hypothesis | No complex organic compounds will be produced. |
| Results | Samples from solution contained formaldehyde, hydrogen cyanide, and several complex compounds with C=C bonds, like amino acids (e.g. glycine). |
| Conclusion | Chemical evolution occurs readily if simple molecules with high free energy are exposed to a source of kinetic energy. |
| Significance | Paved the idea for foundation for ideas of chemical evolution and sparked further experiments. |


### Herschey-Chase Experiment

| Aspect | Description |
| --- | --- |
| Question | Do viral genes consist of DNA or protein? |
| DNA Hypothesis | T2 virus genes consist of DNA. |
| Protein Hypothesis | T2 virus genes consist of protein. |
| Experimental Setup | Label viruses (grow one set of TS w/ radioactive DNA and radioactive protein). Infect bacteria. Agitate cultures. Centrifuge solutions and force cells into pellet. Record location of radioactive labels. |
| Prediction of DNA Hypothesis | Radioactive DNA will be located within the pellet of the centrifuge. |
| Prediction of Protein Hypothesis | Radioactive protein will be located within the pellet of the centrifuge. |
| Results | Radioactive DNA is in pellet. Radioactive protein is in solution. |
| Conclusion | T2 virus genes consist of DNA. |
| Significance | Proved that heritable information takes the form of DNA, not proteins. |

### Meselson-Stahl Experiment

| Aspect | Description |
| --- | --- |
| Question | Is replication semiconservative, conservative, or dispersive? |
| Hypothesis 1 | Replication is conservative. |
| Hypothesis 2 | Replication is semiconservative. |
| Hypothesis 3 | Replication is dispersive. |
| Experimental Setup | Grow *E. coli* cells in medium with `15_N` (heavy nitrogen, see [DNA Replication Terms and Concepts](#dna-replication-terms-and-concepts)). Transfer cells to medium with `14_N` (regular nitrogen). Let the cells divide twice (total 3 generations). Centrifuge the three samples separately. Compare locations of DNA bands.
| Prediction 1 | After two generations: 1/2 low-density DNA, 1/2 intermediate-density DNA. |
| Prediction 2 | 1/4 high-density DNA, 3/4 low-density DNA. |
| Prediction 3 | All intermediate-density DNA. |
| Results | After 2 generations, 1/2 low-density DNA and 1/2 intermediate-density DNA. |
| Conclusion | Replication is semiconservative. |
| Significance | Proved that, in accordance with Watson and Crick's suspicions, that DNA replicated by splitting down the middle and using each half as a template. Thus, each DNA is half-old and half-new. |

### Srb-Horowitz Experiment

| Aspect | Description |
| --- | --- |
| Question | What do genes do? |
| Hypothesis | Each gene contains information needed to make one enzyme. |
| Null Hypothesis | Genes do not have a one-to-one correspondence with enzymes. |
| Experimental Strategy | Produce mutants unable to synthesize arginine, then test different steps in the mtabolic pathway for synthesizing arginine. |
| Experimental Setup | Isolate *Neurospora crassa* that cannot synthesize arginine in four environments: no supplement, supplemented with only ornithine, supplemented only with citrulline, supplemented only with arginine |
| Prediction of Hypothesis | There will be 3 distinct types of mutants corresponding to defects in enzyme 1, 2, and 3 in the pathway for synthesizing arginine. Each mutant will be able to grow  on different combinations. |
| Prediction of Null Hypothesis | There will not be a simple correspondence between a particular mutation and a particular enzyme. |
| Results | Three types of mutants *arg1*, *arg2*, *arg3*. They are able to grow in the metabolic pathway *from* orthinine, citrulline, and arginine, respectively. *arg1* cells lack enzyme 1 from precursor to orthinine, *arg2* cells lack enzyme 2 from orthinine to citrulline, etc. |
| Conclusion | The one-gene, one-enzyme hypothesis is supported. |

### Zamecnik et al Experiment
| Aspect | Description |
| --- | --- |
| Question | What happens to amino acids attached to tRNAs? |
| Hypothesis | Aminoacyl tRNAs transfer amino acids to growing polypeptides. |
| Null Hypothesis | Aminoacyl tRNAs do not transfer amino acids to growing polypeptides. |
| Experimental Setup | Attach radioactive leucine molecules to tRNAs. Attach aminoacyl tRNAs to an in vitro system that allows protein synthesis. Follow the path of radioactive amino acids. |
| Prediction of Hypothesis | Radioactive amino acids will be found in the polypeptides. |
| Prediction of Null Hypothesis | Radioactive amino acids will not be found in the polypeptides. |
| Results | Radioactive signal of tRNA and polypeptides are inversely proportional to each other. When there is low radioactive signal in tRNA, there is high radioactive signal in the polypeptides, and vice versa. |
| Conclusion | Aminoacyl tRNAs transfer amino acids to growing polypeptides. |

[Back to top](#)













[Back to top](#)
