[Home](https://andre-ye.github.io) > [Biology Navigation](https://andre-ye.github.io/biology/biology_navigation) > [Biology Content](https://andre-ye.github.io/biology/biology_navigation#biology-content) > Module 1: Gene Expression, Development, Technologies > Chapter 19 Textbook Notes

# Textbook Notes: Chapter 19, "Control of Gene Expression in Eukaryotes"
Andre Ye, 1/7/21

---

## Navigation
- [Introduction](#introduction)
- [19.1: Gene Regulation in Eukaryotes - An Overview](#191-gene-regulation-in-eukaryotes-an-overview)
  * [Summary of Levels of Gene Expression in Eukaryotes](#summary-of-levels-of-gene-expression-in-eukaryotes)
- [19.2: Chromatin Remodeling](#192-chromatin-remodeling)
  * [Chromatin Structure](#chromatin-structure)
  * [Chromatin Structure is Altered in Active Genes](#chromatin-structure-is-altered-in-active-genes)
  * [How is Chromatin Altered?](#how-is-chromatin-altered)
    + [DNA Methylation](#dna-methylation)
    + [Histone Modification](#histone-modification)
    + [Chromatin-Remodeling Complexes](#chromatin-remodeling-complexes)
  * [DNA and Chromatin Modifications Vary and Can Be Inherited](#dna-and-chromatin-modifications-vary-and-can-be-inherited)
    + [Long-Lasting Epigenetic Marks Imposed During Development](#long-lasting-epigenetic-marks-imposed-during-development)
    + [Epigenetic Inheritance is Widespread](#epigenetic-inheritance-is-widespread)
- [19.3: Initiating Transcription](#193-initiating-transcription)
  * [Promoter-Proximal Elements Are Regulatory Sequences Near the Core Promoter](#promoter-proximal-elements-are-regulatory-sequences-near-the-core-promoter)
  * [Enhancers Are Regulatory Sequences Far from the Core Promoter](#enhancers-are-regulatory-sequences-far-from-the-core-promoter)
  * [The Role of Transcription Factors in Differential Gene Expression](#the-role-of-transcription-factors-in-differential-gene-expression)
  * [How Do Transcription Factors Recognize Specific DNA Sequences?](#how-do-transcription-factors-recognize-specific-dna-sequences)
  * [A Model for Transcription Initiation](#a-model-for-transcription-initiation)

## Introduction
- Bacteria regulate gene expression b/c of changes in their environment.
- However, for eukaryotes, different types of cells must be maintained using the same set of genes.
  - All cells differ from each other, but have the same genes.
- Eukaryote cells must react to changes in the external  *and internal* environments.
- **Differential gene expression** - cells with the same genome can express different sets of genes, because cells at different times and locations are exposed to different signals.

## 19.1: Gene Regulation in Eukaryotes - An Overview
- Eukaryotes can control gene expression at *transcription, translation,* and *post-translation*.
  - There are, however, *more levels of control*.
- **Chromatin** - the sturcture formed by DNA wrapped around proteins.
  - Eukaryotic genes have promoters, but *before* transcription can begin, DNA containing the promoter needs to be *released* from interactions w/ the protein.
  - This allows the RNA polymerase to contact the promoter.
  - This is **chromatin remodeling**.
- **RNA processing** is required to produce a mature mRNA in eukaryotes (splicing introns, etc.)
  - Alternative splicing can include exons, which result in different gene products.
- **mRNA stability** can be regulated, such that certain mRNAs can have a shorter or longer life span.

### Summary of Levels of Gene Expression in Eukaryotes
1. Chromatin remodeling - "opening" chromatin for the RNA polymerase
2. Transcription - produce the pre-mRNA
3. RNA processing - process the pre-mRNA to form mature mRNA
4. mRNA stability - make changes to the mRNA that can degrade mRNA life span, if necessary.
5. Translation - create a polypeptide corresponding to the mRNA sequence.
6. Post-translational modification - activate or deactivate proteins.

## 19.2: Chromatin Remodeling
- For a signal to trigger transcription of a gene, chromatid around the target gene must be remodelled.
- DNA is very tightly wrapped since there is so much information; this does not allow RNA polymerase to access it.
- It must be "unwrapped" such that it *can* be accessed and transcribed.

### Chromatin Structure
- Eukaryotic DNA is associated with **histone** proteins (generally speaking; other proteins may be used as well).
- Chromatin looks like beads on a string; these beads are **nucleosomes**.
  - Each nucleosome is about 200 base pairs of DNA wrapped almost twice around 8 histones.
  - Histone protein called **H1** "seals" DNA to each set of histone proteins.
  - DNA is negatively charged (phosphate groups); histones are positively charged (lysines and arginines).
- Nucleosomes are linked to each other to form chromatin.
  - H1s interact with each other and other histones to produce a tightly packed structure.
- These fibers are attached at intervals along their length to framework proteins; this allows the chromosome to be organized.
- Each chromosome lies in its own territory of the interphase nucleus.
  - Parts fold into actively transcribed sites; location matters.
- Chromatin has important implications for gene expression control.

### Chromatin Structure is Altered in Active Genes
- The interaction between DNA and histones **must be altered** for RNA polymerase to interact with it.
  - A gene cannot be transcribed *until* the condensed chromatin near the promoter is remodeled.
  - **Chromatin remodelling is the first step of eukaryotic gene expression.**
- **DNase** is an enzyme that cuts DNA at random locations. However it cannot cut DNA if it is wrapped with proteins; so it can only work with DNA if the DNA is "decondensed" or in *open configuration*.
  - Wintraub and Groudine experiment: compared chromatin structure of beta-globin and ovalbumin.
  - Beta-blogin is hemoglobin found in red blood cells, ovalbumin is a protein found in egg white. Beta-globin should be transcribed at high levels in blood cells, and the opposite for ovalbumin.
  - Findings: DNase cut beta-globin gene DNA much more than ovalbumin gene DNA. Hence, chromatin of beta-globin was *decondensed* and chromatin the ovalbumin gene was *condensed*.
- **Chromatin is decondensed in genes that are being transcribed**.
- By default, eukaryotic genes are "turned off" b/c they are normally wrapped tightly in chromatin.

### How is Chromatin Altered?
- There are three ways to remodel chromatin: DNA methylation, histone modification, and chromatin-remodeling complexes.

#### DNA Methylation
- **DNA methyltransferases** add methyl groups to cytosine residues in DNA via **DNA methylation**.
- The sequence recognized by enzymes is a C next to a G (`ATAT[CG]TA`). The methylated sequences are abbreviated `CpG`.
- Methylated `CpG` sequences are recognized by proteins that **trigger chromatin condensation**.
  - Hence, actively transcribed genes **have fewer methylated CpG sequences near their promoters**.

#### Histone Modification
- Many enzymes add several chemical groups to histone protein amino acids.
  - Includes acetyl groups, methyl gruops, phosphate groups, and short polypeptides.
- Modifying histones alters the bond between DNA and histone proteins.
  - **This can condense or decondense chromatin**, depending on the modifications.
- Proposal: histone modification combinations set the state of chromatin condensation for a particular gene. **Histone code hypothesis**.
  - **histone acetyltransferases (HATs)** add acetyl groups to positively charged lysine residues of histones. This interferes with assembly nucleosomes in condensed chromatin.
    - Process known as **acetylation** - promotion of *decondensed* chromatin and active transcription. The "on switch".
  - **histone deacetylases (HDACs)** remove acetyl groups. This reverts chromatin into a condensed state. The "off switch".
  
#### Chromatin-Remodeling Complexes
- **Chromatin-remodeling complexs** are proteins that form macromolecular machines.
  - Harness energy in ATP to *reshape* chromatin.
  - Can cause nucleosomes to slide along DNA or knock histones off DNA to open chromatin for transcription.
- **Condensation state of chromatin is critical in determining whether transcription can occur or not.**

### DNA and Chromatin Modifications Vary and Can Be Inherited
- Chromatin modifications vary from one cell type to another (e.g. skin and liver cell).
- When skin or liver cells divide, **DNA methylation and histone modification patterns are passed to duaghter cells.**
  - Daughter cells inherit patterns of gene expression via **epigenetic inheritance**. (inheritance of anything other than DNA sequences)
- Chromatin is modified early in its development to form a certain cell, like a skin cell.
- Epigentic mechanisms can record life events and are difficult to erase. Prenatal conditions can make long-lasting alterations in the chromatin of embryos that influence phenotypes.

#### Long-Lasting Epigenetic Marks Imposed During Development
- Experiment: what happens to rats born to mothers fed low-protein diets during pregnancy and nursing?
  - Animals have a greater risk of developing disorders similar to type 2 diabetes, even though a normal diet was provided after early deprivations.
- *Hnf4a* is a gene associated with diabetes; codes for a regulator of genes involved in glucose uptake.
  - Epigentic inheritence could be silencing *Hnf4a* expression.
- Results of analysis: **histone modifications of *Hnf4a* gene led to condensed chromatin** for rats born to malnourished mothers.
- A mother's nutritional status is repsonsible for chromatin modifications seen in offspring.

#### Epigenetic Inheritance is Widespread
- Environmental influences early in life can be recorded into patterns of chromatin condensation.
- Human mental retardation - Rett syndrome. Occurs when mutation l eads to absence of a protein that binds to methylated DNA to induce chromatin condensation.

## 19.3: Initiating Transcription
- The promoter in eukaryotes is where DNA and RNA polymerase bind to initiate transcription.
- **Core promoter** refers to the sequence RNA polymerase binds to.
  - **TATA box** is an intensively studied core promoter sequence.
- THe **TATA-binding protein (TBP)** binds to initiate transcription.
  - TBP is in a large protein complex.

### Promoter-Proximal Elements Are Regulatory Sequences Near the Core Promoter
- **Regulatory** sequences are other DNA sequences *besides the core promoter* that allow control over the initiation of transcription.
  - Regulatory sequences in eukaryotes are like the operator and CAP binding site.
- How do yeast cells control metabolism of galactose?
  - When galactose is absent, *S. cerevisiae* cells produce small amounts of the five galactose-processing enzymes. When galactose is present, the opposite is true.
  - The regulatory protein binds to a regulatory sequence just upstream from the core promoter for *each* of the five genes.
- **Instead of being clustered into a single operon, co-regulated genes in eukaryotes have the *same* regulatory DNA sequence that binds to the same type of regulatory protein.**
  - These regulatory sequences exist in all eukaryotic genes.
- Regulatory sequences close to the promoter are **promoter-proximal elements**.
  - Different types of promoter-proximal elements are associated with each gene; eukaryotic cells can express certain genes but not others.

### Enhancers Are Regulatory Sequences Far from the Core Promoter
- Researchers found a regulatory sequence required for enhanced transcription in one of the *introns*.
  - The regulatory sequence was thousands of bases away and downstream from the promoter.
  - These regulatory sequences far from the promoter (and can be in nontranscribed sequences) that activate transcription are **enhancers**.
  - Enhancers *occur in all eukaryotes*.
- Enhancers are composed of many short regulatory sequences that bind to a different specific regulatory protein.
- Enhancers work even if they are flipped from their normal orientation or moved to new locations near the gene.
- **Transcriptional activators** (or just **activators**) bind to enhancers to begin transcription.
  - Enhancers and activators are *positive control*, like a gas pedal.
- Regulatory sequences like enhancers that instead inhibit transcription re **silencers**.
  - When **repressors** bind to silencers, transcription is shut down.
  
### The Role of Transcription Factors in Differential Gene Expression
- Enhancers and silencers are binding sites for activators and repressors that regulate transcirption; these proteins are **regulatory transcription factors**.
  - Hundreds of transcription factors binde to enhancers, silencers, and promoter-proximal elements.
- Different types of cells **express different genes b/c of different transcription factors**.
  - Genes encoding transcription factors are expressed inr esponse to signals from other cells (esp. during embryonic development).
- Formation of a muscle cell:
  1. As a cell in the early embryo, production of muscle cell-specific transcription factors begins.
  2. Transcription factors bind to regulatory sequences and turn "on" production of muscle-specific proteins.
- **Differential gene expression results primarily from production or activation of specific transcription factors.**

### How Do Transcription Factors Recognize Specific DNA Sequences?
- Transcription factors must be able to recognize and bind to a DNA sequence.
  - DNA bases are partially exposed in the grooves of the DNA double helix. (major and minor grooves)
- Differences in chemical composition and shape of groves are recognized by transcription factors.
- A transcriptional factor important for development of muscle cells inserts amino acid side chains into the grooves of DNA.
  - Complementary interactions between amino acids and a particular sequence of base pairs in DNA.

### A Model for Transcription Initiation
- Gene expression can be controlled at many levels, *but regulating the start of transcription is perhaps the most important.*
- **General transcription factors** interact w/ the core promoter and are not restricted to *particular genes* or cell types. 
  - "general": proteins are necessary for transcription to occur, but do not provide much regulation.
- TBP is a part of a large general transcription factor.
- The **mediator** acts as a bridge between regulatory transcirption factors, general transcription factors, and RNA polymerase II.
  - Plays an important role of integrating regulatory trnascription factors; delivers a signal to RNA polymerase for transcription initiation.
- Process:
  1. Activators bind to DNA. Chromatin-remodeling complexes and histone acetyltransferases are used, and chromatin decondenses.
  2. Some chromatin that is exposed includes the core promoter, promoter-proximal elements, and enhancers.
  3. Other activators bind to exposed enhancers and promoter-proximal events. DNA-bound activators bind to the Mediator.
  4. General transcription factors and RNA polymerase II assemble to the Mediator and associate with the core promoter.
  5. DNA strands are opened, RNA polymerase II begins transcription.
- Activators work to stimulate transcription, but *also* to bring chromatin-remodeling proteins to the right place.
- Chromatin remodeling proteins cannot recognize specific DNA sequences; transcriptional activators bind first to recruit them.
- DNA sometimes disassociates from histone proteins in nucleosomes, exposing regulatory sequences to activators.

[Back to top](#)
