[Home](https://andre-ye.github.io) > [Biology Navigation](https://andre-ye.github.io/biology/biology_navigation) > [Biology Content](https://andre-ye.github.io/biology/biology_navigation#biology-content) > Study Guides > Biology Cheat Sheet / Reference

## Biology Cheat Sheet / Reference

*This biology cheat sheet will be handy reference for assignments, discussions, and test prep. It will be continually updated as the course progresses.*

<br>

---

<br>

### Navigate
- [Terms and Concepts](#terms-and-concepts)
  * [Control of Gene Expression in Bacteria](#control-of-gene-expression-in-bacteria)
  * [Control of Gene Expression in Eukaryotes](#control-of-gene-expression-in-eukaryotes)
  - [Mendelian Genetics](#mendelian-genetics)
- [Core Ideas](#core-ideas)
  * [Table of Elements of Transcriptional Regulation](#table-of-elements-of-transcriptional-regulation)
  * [Terms Used in Mendelian Genetics](#terms-used-in-mendelian-genetics)
  
<br>

---

<br>

### Terms and Concepts

#### Control of Gene Expression in Bacteria
From [Textbook Notes: Chapter 18, “Control of Gene Expression in Bacteria”](https://andre-ye.github.io/biology/winter-quarter/notes/module-1/chapter-18){:target="_blank"}

| Term | Definition |
| --- | --- |
| allolactose | A variant of lactose that acts as an inducer of transcription, binding to the lac repressor to prevent it from blocking RNA polymerase. |
| cAMP | Cyclic AMP allows CAP to bind and promote transcription in the lac operon. |
| catabolite activator protein (CAP) | A protein that exhibits positive control on many operons in *E. coli* and is transcribed & transcripted constitutively. CAP binds to a regulatory sequence of DNA and increases the frequency of initiating transcription by promoting the association of the RNA polymerase holoenzyme w/ the promote. |
| catabolite repression | Catabolite repression is an important part of the control system of bacteria; it inhibits synthesis of enzymes involved in the catabolism of carbon sources other than the preferred one. For instance, glucose initiated catabolite repression. |
|	constitutive | A gene is transcribed *constitutively* if it is done so without regards for gene expression control, like enzymes required for glycolysis. |
|	constitutive mutants | Cells that produce a product consistently at all times, instead of regulating its expressions. |
|	disaccharide | A sugar with two monosaccharide residues. For instance, lactose is a disaccharide, consisting of galactose and glucose. |
|	galactose | The monosaccharide sugar that, with lactose, forms lactose. |
|	gene expression | The "expression" of genes, or the transfer of information into the genotype into physical qualities of the phenotype. |
|	genetic screen | A technique used to identify and select individuals with a particular phenotype in a population with mutants. (Rephrased: Any technique that identifies individuals with a particular type of mutation.) |
|	global gene regulation | Bacteria may need to coordinate change globally instead of individual genes or operons. Global gene regulation is the coordinated regulation of many genes, across operons. |
|	glucose monosaccharide | Glucose is a monosaccharide sugar that is required for cellular respiration to produce ATP. Together with galactose, it forms the disaccharide lactose. |
| inducer | An inducer is a small molecule that triggers the transcription of a gene. It can disable repressors or bind to activators, and therefore allows transcription to start. |
| lac operon | Genes involved in lactose metabolism form the lac operon. |
| `lacI` | A gene that prevents the transcription of `lacZ` and `lacY` when lactose is absent. It acts as a negative regulator, and lactose acts as an inducer. `lacI` produces a repressor protein that binds to DNA and overlaps the promoter for `lacZ` and `lacY` genes, preventing RNA polymerase from transcribing. However, lactose binds to it and changes its shape, causing this repressor to release from its binding site. |
| `lacI-` | A lactose metabolism mutant that constitutively expresses *lacZ* and *lacY* because the gene for the regulatory protein that shuts down *lacZ* and *lacY* is defective. |
| lactose | A dissacharide sugar that can be broken down into the monosaccharide sugars galactose and glucose. |
| `lacY` | A gene part of the lac operon that codes for galactosidase permease, which transports lactose into the cell. |
| `lacY-` | A lactose metabolism mutant that cannot accumulate lactose. It lacks galactoside permease to bring lactose into the cell because the *lacY* gene is defective. |
| `lacZ` | A gene part of the lac operon that codes for beta-galactosidase permease, which breaks lactose into glucose and galactose. |
| `lacZ-` | A lactose metabolism mutant that cannot cleave lactose even in the presence of the inducer. It lacks beta-galactosidase because the *lacZ* gene is defective. |
| medium | The environment in which an organism is placed in. |
| mutagens | A substance or agent that induces heritable change in cellular organisms. |
| negative feedback control | When the final product of a pathway inhibits the production of a product. See the trp operon for an example. |
| operator | The DNA sequence that the repressor binds to. |
| operon | A set of coordinately regulated bacteria genes transcribed into one polycistronic (codes for multiple polypeptides) mrNA. |
| permease | Enzymes that transport, like galactosidase permease, which transports lactose into the cell. }
| positive feedback | The opposite of a negative feedback control; when the final product of a pathway promotes the production of a product. |
| post-translational control | Regulation of activated proteins, for instance by preventing protein activation steps like the addition of a phosphate group. This is an energetically cheap action, but inefficient because the entire protein needs to be constructed. |
| regulon | A set of separate genes and operons that contain the same regulatory sequences, and are controlled by one type of regulatory protein. Regulons are used in global gene regulation to coordinate regulation of many genes. |
| repressor | A regulatory protein that binds to DNA at the operator and stops transcription. |
| transcriptional control | A level of gene expression control in which certain genes are not transcribed into mRNA (or pre-mRNA for eukaryotes). |
| translational control | A level of gene expression control in which certain mRNA are not translated into proteins, for instance by degrading mRNA or reducing its lifespan.
| trp operon | A form of negative control feedback, the trp operon is used to synthesize tryptophan (an aminoa cid). Genes for trptophan synthesis are expressed only when tryptophan is absent, so the trp repressor - which stops transcription - binds to the operator *only* when it is bound by tryptophan, its regulator. |

<br>

#### Control of Gene Expression in Eukaryotes
From [Textbook Notes: Chapter 19, “Control of Gene Expression in Eukaryotes”](https://andre-ye.github.io/biology/winter-quarter/notes/module-1/chapter-19){:target="_blank"}

| Term | Definition |
| --- | --- |
| acetylation | The promotion of *decondensed* chromatin and activation of transcription. As this allows chromatin to be unwound, it is the "on switch". HATs add acetyl groups to positively charged lysine residues of histones, interfering with the assembly of nucleosomes in condensed chromatin. |
| activator | Transcriptional activators bind to enhancers to begin transcription. THey are *positive control*. |
| alternative splicing | Alternative splicing can include introns, resulting in different gene products. This can alter the stability of mRNA, such that certain mRNA has a shorter or longer life span (is more or less easily degraded, respectively). |
| chromatin | The structure formed by DNA wrapped around proteins, often histones. |
| chromatin remodeling | Before transcription can begin in eukaryotes, DNA containing the promoter must be released from interactions with the protien, allowing RNA polymerase to contact the promoter. The process of doing this is chromatin remodeling. |
| DNA methylation | DNA methyltransferases add methyl groups to cytosine residues in DNA, forming sequences abbreviated `CpG`. These methylated `CpG` sequences are recognized by proteins that trigger chromatin condensation ('tightening'). Actively transcribed genes have fewer methylated `CpG` sequences near their promoters, such that the chromatin is less tightly wound. |
| enhancer | Enhancers are regulatory sequences far away (i.e. thousands of bases) from the promoter, but nevertheless activate transcription. Enhancers are composed of many short regulatory sequences that allow for the binding of a different specific activator. Enhancers work even if they are flipped from their normal orientation, or moved to new locations near the gene. Enhancers occur in all eukaryotes, and can be located in nontranscribed as well as transcribed sequences. |
| epigenetic inheritance | Cells have patterns of gene expression; daughter cells inherit this pattern through epigenetic inheritance. This way, daughter cells of a skin cell are also skin cells. Epigenetic mechanisms can record life events and are difficult to erase; prenatal conditions, for instance, can cause long-lasting changes in the chromatin of embyros, influencing phenotypes. |
| exon | Part of the pre-mRNA of eukaryotes that are retained and spliced into the mature mRNA. |
| histones | Histones are proteins around which DNA often wraps around to form chromatin.  Modifying histones alters the bond between DNA and histone proteins, and can condense (tighten) or decondense (loosen) chromatin, depending on the modifications. |
| histone acetyl transferases (HATs) | HATs add acetyle groups to positively charged lysine residues of histones, interfering with the assembly of nucleosomes in condensed chromatin, through acetylation. This initiates transcription. |
| histone deacetylases (HDACs) | HDACs remove acetyle groups, and revert chromatin into a condensed state by tightening it. This is like the "off switch" of transcription. |
| intron | Part of the pre-mRNA of eukaryotes that are spliced out of the mature mRNA. |
| mediator | The mediator is a bridge between regulatory transcription factors, general transcription factors, and RNA polymerase II. It integrates regulatory transcription factors and delivers a signal to RNA polymerase for transcription initiation. |
| methylation | See DNA methylation. In general, methylation is the addition of a methyl group to a substrate. |
| phosphorylation | Phosphorylation is a post-translational step. In general, it is the addition of a phosphate group to a substrate. |
| post-translational regulation | Regulation of activated proteins, for instance by preventing protein activation steps like the addition of a phosphate group. This is an energetically cheap action, but inefficient because the entire protein needs to be constructed. |
| regulatory sequences | In eukaryotes, regulatory sequences are DNA sequences that allow control over the initiation of transcription *other than* the core promoter. Regulatory seuqences are like the operator and CAP binding sites in prokaryotes. |
| repressors | A regulatory protein that binds to DNA at the operator and stops transcription. |
| RNA processing | A step that occurs in the Central Dogma for eukaryotes, in which introns are spliced out, among other changes. |
| silencer | Sharing the same physical characteristics of an enhancer (i.e. far away from the promoter), the silencer instead allows for repressors to bind. When repressors bind to silencers, transcription is inhibited. |
| transcription factors | Regulatory transcription factors are proteins acting as activators and repressors that bind to enhancers and silencers. General transcription factors interact with the core promoter and are not restricted to any particular gene or cell types; these factors do not provide regulation. |
| transcriptional activator | Activators that bind to enhancers to begin transcription. These must bind to enhancers first in order to recruit chromatin remodeling proteins (which cannot recognize specific DNA sequences). |

<br>

#### Mendelian Genetics

| Term | Definition |
| --- | --- |
| alleles | A "version" or "copy" of a gene. |
| autosomes | Chromosome pairs that are not sex chromosomes. |
| blending inheritance | The hypothesis that the traits of a mother and father blend together to form traits in offspring. |
| carrier | An individual who carries and is capable of passing on some genetic trait. |
| crossing over | A step in meiosis in which nonsister chromatids of homologous chromosomes can randomly swap, or "cross over" part of their allele. |
| dihybrid cross | A cross between two organisms (dihybrids) that differ in two traits. |
| diploid | A cell is diploid when it has two alleles for each gene. |
| dominance (dominant) | A dominant allele type is phenotypically expressed over a recessive allele type. |
| F1 generation | Offspring (or progeny) of the parental generation - the first cross. |
| F2 generation | Offspring of the F1 generation - the second cross. |
| gene | A stretch of DNA on a chromosome that determines a trait. |
| genotype | The genetic information of an individual. |
| heredity | Inheritance of traits. |
| heterozygous | An organism that has different alleles for a trait. |
| homozygous | An organism that has the same alleles for a trait. |
| hybrids | Offspring with matings between true-breeding parents that differ in one or more traits. |
| monohybrid cross | A cross between two organisms that differ in a single given trait. |
| mutant | An individual with a heritable change in a gene. |
| parental generation (P1) | The initial organisms that are crossed to form the F1 generation. |
| phenotype | The expressed traits from the genotype. |
| principle of independent assortment | Alleles for genes separate completely randomly into gamets, irrespective of alleles for other genes. |
| Punnett square | A tool for understanding outcomes of genetic crosses. |
| pure lines | Individuals that produce offspring identical to the parents when they are crossed to another member of the same population or are self-fertilized. |
| recessiveness (recessive) | An allele that is only expressed when there is no dominant allele for that gene present. |
| reciprocal cross | When the male and female gametes are swapped and crossed. This is used to ensure that traits are not sex-linked and independently assorted. |
| recombination (recombinants) | The rearrangement of DNA sequences by the breakage and rejoining of chromosomes or chromosome segments. |
| sex chromosomes | Chromosomes associated with the offspring's sex; chromosome 23 in humans. |
| sex-linked inheritance | A trait associated with sex because it is located on sex chromosomes. |
| testcross | A cross between a phenotypically dominant (but unknown genotype) organism with a genotypically recessive organism to determine the former's genotype. |
| trait | An observable characteristic about an individual. |
| wild type | The naturally occuring non-mutant allele of a gene. |
| X chromosome | The chromosome females have two of, and males have only one of. |
| y chromosome | The chromosome females have none of, and males have one of. |

<br>

---

<br>

### Core Ideas
#### Table of Elements of Transcriptional Regulation
![](https://raw.githubusercontent.com/andre-ye/andre-ye.github.io/master/biology/winter-quarter/study-guide/images/elements-of-transcriptional-regulation.png)

#### Terms Used in Mendelian Genetics
![](https://raw.githubusercontent.com/andre-ye/andre-ye.github.io/master/biology/winter-quarter/notes/module-2/images/mendelian_genetics_terms.png)

<br>

---

<br>

[Back to top](#)

